hadoop/README.md

```diff
@@ -11,7 +11,9 @@ installed.
 
 2. [Apache Maven](https://maven.apache.org/)
 
-3. Tested with Hadoop 2.6.0. Patches are welcome if there are incompatibilities
+3. Downloads protos from [googleapis](https://github.com/tensorflow/tensorflow)
```

Comment:
How about `3. Download and compile protos from [TensorFlow](https://github.com/tensorflow/tensorflow).`